### DISCUSSION TITLE
how to run imgui/egui example on windows xp

### DISCUSSION THREAD

**Discussion by xixioba** [msys2/MINGW-packages#22109] [Q&A]
I setup the **msys2 mingw32** environment (**version 240727**) and compiled rust 1.76 for **windows xp** and tested that the c++/rust helloworld program can run correctly under windows xp.

Next, I tested the sample code of [imgui](https://github.com/ocornut/imgui) and [egui](https://github.com/emilk/egui) respectively and fixed some potential problems, but unfortunately, they all ended up with the problem that `point_ftol2 could not be located int the dynamic link library msvcrt.dll`

Here are the relevant links in the two repositories:
- https://github.com/ocornut/imgui/discussions/8039
- https://github.com/emilk/egui/discussions/5172

Any help is greatly appreciated!

**Comment by MehdiChinoune** [comment]
Windows XP was dropped a long time ago.
